src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -27,64 +26,59 @@ const defaultProps = {
     onfidoToken: null,
 };
 
-class RequestorOnfidoStep extends React.Component {
-    constructor(props) {
-        super(props);
-        this.submit = this.submit.bind(this);
-    }
+const stepCounter = {step: 3, total: 5};
```

Comment:
Since, it is a constant outside of a component, usually we use upper-snake-case.
```suggestion
const STEP_COUNTER = {step: 3, total: 5};
```

Reply:
@rezkiy37 UPER_SNAKE_CASE isn't reserved for global constants?

Reply:
Hmmm, I'm not sure that we have such reservation. I used the case for local constants.
cc @gedu @fabioh8010

Reply:
I think there isn't a style guideline about this, but I have seen variables like this one declared outside the component with UPER_SNAKE_CASE format.

Reply:
Maybe Michael's code 😄  ok, so I'll go with it.